locale/en/common.po

```diff
@@ -160,6 +160,12 @@ msgstr "The editors express their appreciation of the reviewers for {$year} list
 msgid "common.editorialHistory"
 msgstr "Editorial History"
 
+msgid "common.editorialHistory.page"
+msgstr "Editorial History Page"
+
+msgid "common.editorialHistory.page.description"
+msgstr "Here, you will find a comprehensive record of all past contributors associated with this journal. This page highlights our commitment to transparency and showcases our editorial evolution."
```

Comment:
Locale keys in `pkp-lib` shouldn't reference OJS-specific language, and I think the style is a little too advertorial for a platform. I think it's OK for us to be activist in the PFL, but this is built into OJS/OMP/OPS. I suggest we either use something neutral:

> This area provides a comprehensive record of past contributors.

...or if that's not helpful to OJS/OMP/OPS users in general, that we don't include a description.

Reply:
OK, I would change it like that, if @Devika008 has nothing against it?

Reply:
If it would be me, I would not have any description on this page -- I am a minimalist regarding descriptions/explanations -- only just really needed :sweat_smile:

Reply:
We can make it really simple with

Heading: View Editorial History
Description Text: This section lists past contributors.

My intention of including a hint text below the description is to set the intention of the page for the reader since these are new additions. If you both feel that its not necessary we can really get away with it.